iTunesUDataPurge/config.properties.template

```diff
@@ -0,0 +1,3 @@
+iTunesUSiteId=iTunesU_site_id
+sharedSecret=iTunesU_site_sharedSecret
+admin_credential=admin_credential
```

Comment:
Fixed. config.properties.template now has the same parameter settings as those on deluxe server.